src/queries.js

```diff
@@ -98,3 +98,9 @@ export const mkHomeCustomShorcutsConn = foldersId => {
     fetchPolicy: defaultFetchPolicy
   }
 }
+
+export const fetchAppInfo = async (appId, client) => {
+  const appQuery = Q('io.cozy.apps').getById(appId)
+  const { data } = await client.query(appQuery, { as: 'fetchAppInfoReady' })
```

Comment:
Tu ne peux pas appeler ton alias (`as`) ainsi parce que tout le cache de cozy-client est basé sur ce `as`... 

https://github.com/cozy/cozy-guidelines/#naming-of-queries

Reply:
Corrigé ✅